### DISCUSSION TITLE
How can I find server ip at server part?

### DISCUSSION THREAD

**Discussion by dolgae03** [ANLAB-KAIST/KENSv3#101] [Project Q&A 1. Application Layer (Socket)]
I can't find server ip with any method. it always return 0.0.0.0. and i don't know why It happened.
I used getsockname method with bind socket. Is there any wrong usage in command?

![image](https://user-images.githubusercontent.com/109707681/190412079-b3ca367c-9f9f-4338-9416-7989170fac55.png)

**Comment by whkim0** [comment]
`0.0.0.0` is an address the server's socket has `bind()` to, not the server's IP address in the connection between the client and server.
https://stackoverflow.com/questions/20778771/what-is-the-difference-between-0-0-0-0-127-0-0-1-and-localhost

P.S. I don't recommend posting your own code, as I don't know the context and we are not going to debug your code. Also, you are exposing your own work. 😄

**Reply by dolgae03** [reply]
Thank you!
